src/DataGrid.tsx

```diff
@@ -306,15 +337,39 @@ function DataGrid<R, K extends keyof R, SR>({
       if (!current) return;
       current.scrollTop = rowIdx * rowHeight;
     },
-    selectCell(position: Position, openEditor?: boolean) {
-      eventBus.dispatch('SELECT_CELL', position, openEditor);
-    }
+    selectCell
   }));
 
   /**
    * event handlers
    */
-  function onGridScroll(event: React.UIEvent<HTMLDivElement>) {
+  function handleKeyDown(event: React.KeyboardEvent<HTMLDivElement>) {
+    if (enableCellCopyPaste && isCtrlKeyHeldDown(event)) {
+      // event.key may be uppercase `C` or `V`
+      const lowerCaseKey = event.key.toLowerCase();
+      if (lowerCaseKey === 'c') return handleCopy();
```

Comment:
Not a big fan for this way of writing code. Intuitively I thought `handleCopy` would return something, but it's `void`, so my preference would just to split this into multiple lines.

```typescript
if (lowerCaseKey === 'c') {
  handleCopy();
  return;
}
```

Reply:
I copied the old code. I am okay with either style